Pokepedia.ApiAdapter/PokeApi/GetPokemon.cs

```diff
@@ -22,6 +22,12 @@ public static async Task<PokemonModel> GetPokemonByIdAsync(int id)
             return await GetPokemonAsync(urlQuery);
         }
 
+        public static async Task<PokemonModel> GetLocationByPokemonNameAsync(string name)
```

Comment:
again you arent getting a pokemon. Youre geting the locations for a pokemon. So would this make sense to be inside the getpokemon cs file?